src/Legacy/DynamoRevitVersionSelector/VersionSelectorApp.cs

```diff
@@ -60,15 +62,29 @@ public Result OnStartup(UIControlledApplication application)
             var dynamoProducts = FindDynamoRevitInstallations(debugPath, revitVersion);
 
             Products = new List<DynamoProduct>();
+            PlaylistProducts = new List<DynamoProduct>();
             foreach (var p in dynamoProducts)
             {
                 var path = VersionLoader.GetDynamoRevitPath(p, revitVersion);
                 if (!File.Exists(path))
                     continue;
 
                 Products.Add(p);
+
+                if (p.VersionInfo.Build >= 1 && p.VersionInfo.Major >= 1)
```

Comment:
shouldn't this be `Major` and `Minor` ? https://msdn.microsoft.com/en-us/library/system.version(v=vs.110).aspx#Anchor_6